src/components/db.ts

```diff
@@ -147,9 +147,7 @@ const initResumePreview = async (db: Database): Promise<void> => {
       )
     `,
   );
-  await db.run(
-    'CREATE INDEX IF NOT EXISTS ix_preview_id ON resume_preview_info (preview_id)',
-  );
+  await db.run('CREATE INDEX IF NOT EXISTS ix_preview_id ON resume_preview_info (preview_id)');
```

Comment:
Following how the other indices are named, it looks like the index name should be `ix_resume_preview_info_preview_id`.